main/repl/timerServices.md

```diff
@@ -58,41 +111,26 @@ A `TimerRepeater` has two methods, `schedule()` and `disable`:
     scheduled with this repeater won't be rescheduled again 
     after `E(waker).wake()` is next called on them.
     
-## TimerService methods
-
-### `E(home.<chain or local>TimerService).getCurrentTimestamp()`
-- Returns: A monotonically increasing timestamp for each instance of the chain.
-
-The current block might be executed more than once in case of restart or replay.
-But each time it will start from the same state and receive the same inputs. 
-Since this is repeatable computation, the same computation can be run in various
-locations to cross-verify. So each time, the `currentTimeStamp` will be the same, 
-even if we're running the computation a minute or a month later. It's expressed 
-as a Unix epoch time in milliseconds or similar.
-```js
-command[1] E(home.chainTimerService).getCurrentTimestamp()
-history[1] 1608523721
-command[2] E(home.localTimerService).getCurrentTimestamp()
-history[2] 1340435997
-```
+## Advanced TimerService methods
 
 ### `E(home.<chain or local>TimerService).setWakeup(baseTime, handler)`
 - `baseTime` `{ Timestamp }` 
 - `handler` `{ Handler }`
 - Returns: `{ Timestamp }` 
 
-Calls the specified handler the specified number of seconds after executing this call.
+Calls the specified handler when the current timestamp is at least `baseTime`.
+**NOTE: `baseTime` is an absolute, not relative time.**
 
 Returns the time, in the same format as the parameter,
 at which the call is scheduled to happen. 
 
 ```js
 command[3] handler = harden({ wake: now => { console.log(`woke up ${now}`); }})
 history[3] {"wake":[Function wake]}
-command[4] willWakeAt =  E(home.localTimerService).setWakeup(3, handler)
-history[4] 1342464583
+command[4] willWakeAt =  E(home.localTimerService).setWakeup(1342464580000n + 3_000n, handler)
+history[4] 1342464583000n
```

Comment:
`1342464583000n` is in the range of timestamps I've seen recently from the localTimerService. Contrary to `history[2] 1608523723203n` above.   Why is the local timer producing time stamps that look like milliSeconds since **1979**?

Reply:
> Why is the local timer producing time stamps that look like milliSeconds since **1979**?

Thanks for catching this.  I'll fix that in this documentation.

That was a miscalculation in the code that existed before #3853.